independent-projects/bootstrap/core/pom.xml

```diff
@@ -14,6 +14,10 @@
     <name>Quarkus - Bootstrap - Core</name>
 
     <dependencies>
+        <dependency>
+            <groupId>org.ow2.asm</groupId>
+            <artifactId>asm</artifactId>
+        </dependency>
```

Comment:
Where is this used in core?

Reply:
The ClassLoader implementation allows for transformers, and the transformer API is based on ASM.